docs/docs/how-tos/jupyter-gallery.md

```diff
@@ -0,0 +1,102 @@
+
+In Nebari, the JupyterLab-Gallery configuration is set in the Nebari configuration, `nebari-config.yaml`.
+
+The configuration is set under the `jupyterlab` section under the subsection `gallery_settings`.
+Users may set a title for the Gallery section of the Launcher screen and a destination (i.e. folder)
+for the location of the cloned repos.
+
+Several repositories can be configured to display as individual gallery tiles. Under the `exhibits`
+section, each repository must have a `title` and a `git` location. There are also several other
+optional config options.
+
+The configuration is set at two different levels. Top level gallery settings define the title of
+the Gallery section and the location in which the repositories are downloaded. The repository
+level settings are defined for each repository. The descriptions of each section are described
+below.
+
+**Top Level Gallery Settings**
+
+| Section     | Description                                        |
+| ----------- | -------------------------------------------------- |
+| title       | Title on the gallery tile                          |
+| destination | Location on disk where the repo will be downloaded |
+
+**Repository Level Settings**
+
+| Section     | Description                                                                  |
+| ----------- | ---------------------------------------------------------------------------- |
+| title       | Title on the gallery tile                                                    |
+| git         | URL of the git repository                                                    |
+| homepage    | (Optional)                                                                   |
+| description | Description of the repository to appear on the gallery tile (Optional)       |
+| icon        | base64 encoded image to use an icon to appear on the gallery tile (Optional) |
+| account     | (Optional)                                                                   |
+| token       | Private Access Token to enable access to private repositories (Optional)     |
+
+### Sample configuration
+
+Below is an example of the Gallery settings from the `nebari-config.yaml`.
+
+```
+jupyterlab:
+  gallery_settings:
+    title: Demos
+    destination: demos
+    exhibits:
+    - title: Data of an Unusual Size
+      git: https://github.com/nebari-dev/big-data-tutorial.git
+      description: Big data tutorial focusing on Dask usage in Nebari
+      icon: "data:image/png;base64,<encoding>"
+      homepage: https://github.com/nebari-dev/big-data-tutorial/
+    - title: From RAGs to riches
+      git: https://github.com/Quansight/ragna-presentations.git
+      homepage: https://github.com/Quansight/ragna-presentations/
+    - title: Private Examples
+      description: Internal company examples
+      icon: "data:image/png;base64,<encoding>"
+      git: https://gitlab....net/companyxyz/examples.git
+      account: examples
+      token: "{{ GITLAB_PAT }}"
```

Comment:
Is this just the syntax we use because we are replacing it via CI? Or does jupyter-gallery handle it?

Reply:
> Is this just the syntax we use because we are replacing it via C

Yes.

> Or does jupyter-gallery handle it?

No, jupyter-gallery does not handle variable substitution.

Reply:
Got it. I'll adjust this.